v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v2
 publisher: felixfbecker
-name: php-debug
+name: vscode-php-debug
```

Comment:
AFAIK the existing name is correct: https://github.com/felixfbecker/vscode-php-debug/blob/master/package.json#L2

Reply:
Thanks Florent! Will provide another PR into che-devfile-registry.